test/Microsoft.Extensions.Primitives.Tests/StringValuesTests.cs

```diff
@@ -294,7 +294,11 @@ public void Contains(StringValues stringValues, string[] expected)
 
         [Theory]
         [MemberData(nameof(FilledStringValuesWithExpected))]
-        public void CopyTo_TooSmall(StringValues stringValues, string[] expected)
+        public void CopyTo_TooSmall(
+            StringValues stringValues,
+#pragma warning disable xUnit1026 // Theory methods should use all of their parameters
+            string[] expected)
```

Comment:
Ok, I've remove it. We have other member data that provide the same test coverage, `FilledStringValues`, `DefaultOrNullStringValues`, and `EmptyStringValues`